### ISSUE TITLE
Geki and Katu

### ISSUE THREAD

**Issue by KirbyJoe** [ppy/osu#4625]
**Describe the missing feature:** 
"Geki" and "Katu" are the scoring terms that depends on the accuracy of every comboset. They are missing on both gameplay and result screen.

**Comment by peppy** [comment]
For the time being we have removed this for simplicity. I'm not sure if it will be added back or not.

**Comment by LittleEndu** [comment]
Should this issue be separated into what future lazer skins would support and what legacy skins should support. I see no issue supporting this on legacy skins. 

Gameplay and result screen are also pretty separate things. Even if legacy skins would have visual geki and katu scores, that doesn't mean lazer should display them during results screen.

**Comment by peppy** [comment]
yep, we will definitely add skin support for this, but not scoring (beyond maybe internal HP increase changes).
